package.json

```diff
@@ -17,7 +17,7 @@
     "build:icons": "yarn react build:icons",
     "test": "yarn react test && yarn e2e test",
     "prepare": "husky install",
-    "version:next": "yarn changeset version --snapshot next && yarn build",
+    "version:next": "yarn changeset version --snapshot next-$(git rev-parse --short=7 HEAD) && yarn build",
```

Comment:
Put `--short=7` as length 7 short hash seemed standard in GitHub.

Reply:
For comparison,

https://github.com/aws-amplify/amplify-ui/blob/6a0c6e32dbb253c49291a1d7bc6cbd222a8c5001/docs/next.config.js#L4

yields `623-signup-attributes`, while your change yields `746c0a4`

👍